README.md

```diff
@@ -56,6 +42,21 @@ There is a tiny bridge vault file `vault-env` that bridges the `INFRASTRUCTURE_A
 export INFRASTRUCTURE_ANSIBLE_VAULT_PASSWORD="top secret"
 ```
 
+## Docker image
+
+A Docker image `aeternity/infrastructure` is build and published to DockerHub. To use the image one should configure all the required credentials as documented above and run the container:
+
+```bash
+docker run -it --env-file env.list -v ~/.ssh/:/root/.ssh/ aeternity/infrastructure
```

Comment:
Unfortunately sharing socket (ssh-agent) with docker does not work, thus this is the only working solution I can think of. Any better ideas are welcome.